CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### New Features
```

Comment:
please also include theming tint color options

Reply:
@iaulakh It was added before and this pr is only a name change. Just to confirm, should I add it?

Reply:
Yes @Mrayyan , it will help us to remind theming feature during release.